docs/content/docs/custom-resource/reference.md

```diff
@@ -57,6 +57,23 @@ This serves as a full reference for FlinkDeployment and FlinkSessionJob custom r
 | Parameter | Type | Docs |
 | ----------| ---- | ---- |
 
+### FlinkBlueGreenDeploymentConfigOptions
```

Comment:
I think we should define the Blue Green deployment concept some where in the docs, ideally with diagrams.